go.mod

```diff
@@ -2,13 +2,17 @@ go 1.18
 
 module github.com/cosmos/ibc-go/v6
 
+// Use cosmos-flavored protobufs
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
+// Dragonberry ics23 patch
+replace github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
+
 require (
 	cosmossdk.io/math v1.0.0-beta.3
 	github.com/armon/go-metrics v0.4.1
 	github.com/confio/ics23/go v0.7.0
-	github.com/cosmos/cosmos-sdk v0.46.2
+	github.com/cosmos/cosmos-sdk v0.46.4-0.20221021120231-7f88c60e70d1
```

Comment:
Shouldn't this be simply `github.com/cosmos/cosmos-sdk v0.46.3`?

Reply:
No, still working through some things.  Think it is best for now to work from the last commit in v0.46.*

Reply:
But will this eventually import v0.46.3 before we merge?